frontend/src/app/map-view/scenario/scenario-list/scenario-list.component.ts

```diff
@@ -141,7 +141,37 @@ export class ScenarioListComponent extends Listable implements OnDestroy {
     }
   }
 
-  isDisabled = () => { return !(this.isBatchMode() && this.selectedBatchIds.length > 1) };
+  disableBatch = () => { return !(this.isBatchMode() && this.selectedBatchIds.length > 1) };
+  noneSelected = () => { return !(this.isBatchMode() && this.selectedBatchIds.length > 0) };
+  deleteSelectedScenarios = async () => {
+    if (this.selectedBatchIds.length === 1) {
+      const selectedScenario = await
+        lastValueFrom(this.scenario$.pipe(
+          map(scenarios => scenarios.find(s => s.id === this.selectedBatchIds[0]))
+        ));
+      if (selectedScenario) {
+        await this.deleteScenario(selectedScenario)
+      }
+    }
+    if (this.selectedBatchIds.length > 1) {
+      const confirmDeleteMany = await this.dialogService.open<boolean>(
```

Comment:
Put this in scenario-common.ts as you do with the deleteScenario?

Reply:
It would be unnecessary.
The reason we use the separate scenario-common.ts is that the same method need to be available to both the detailed scenario tab/panel component `scenario-detail.component` and the list view `scenario-list.component`.
Calling a modal for deleting multiple scenarios is, and will likely only ever be required by the list component, really.

I suppose an argument _could_ concievably be made for differentiating the method `deleteScenario`. But I think introducing variability in that method signature will make the Typescript code quite unseemly therein, gaining only a very slight improvement in the model api.